src/components/2022/grid-cell.js

```diff
@@ -10,28 +10,36 @@ const GridCell = (props) => {
   cssProperties['--row-span'] = (props.rowSpan) ? props.rowSpan : null
   cssProperties['--row-start'] = (props.rowStart) ? props.rowStart : null
   cssProperties['--order'] = (props.order) ? props.order : null
```

Comment:
Very minor but all of these could be `cssProperties['--order'] = props.order ?? null` and be shortened up a little.

Reply:
I did not know that! Thank you!